packages/mux-player/src/template.ts

```diff
@@ -57,8 +57,6 @@ export const content = (props: MuxTemplateProps) => html`
     playback-rates="${props.playbackRates ?? false}"
     default-show-remaining-time="${props.defaultShowRemainingTime ?? false}"
     title="${props.title ?? false}"
-    poster="${props.poster === '' ? false : props.poster ?? false}"
-    placeholder="${props.placeholder ?? false}"
```

Comment:
does mux-player-react/lazy use this? If so, it would need to be updated as well.

Reply:
I think this is all under the hood for those pkgs because they interface with the `<mux-player>` WC